services/api-server/tests/unit/pact_broker/conftest.py

```diff
@@ -0,0 +1,61 @@
+import os
+import socket
+
+import pytest
+
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    group = parser.getgroup(
+        "oSparc e2e options", description="oSPARC-e2e specific parameters"
+    )
+    group.addoption(
+        "--broker-url",
```

Comment:
what if we prefix these with `pact`?

this way:
- they are grouped
- they fit as well with the env var names i.e. PACT_BROKER_USERNAME